tests/testthat/_snaps/bootstrap.md

```diff
@@ -0,0 +1,66 @@
+# bootstrapping gives informative output when succesful
+
+    Code
+      bootstrap(version = "0.9.0", library = library)
+    Output
+      Bootstrapping renv 0.9.0:
```

Comment:
I now just output renv + version once, instead of on each line.

Reply:
Makes sense!